spec/features/search_food_spec.rb

```diff
@@ -0,0 +1,41 @@
+# As a user,
+# When I visit "/"
+# And I fill in the search form with "sweet potatoes"
+# (Note: Use the existing search form)
+# And I click "Search"
+# Then I should be on page "/foods"
+# Then I should see a total of the number of items returned by the search.
+# (sweet potatoes should find more than 30,000 results)
+# Then I should see a list of TEN foods that contain the ingredient "sweet potatoes"
+# And for each of the foods I should see:
+# - The food's GTIN/UPC code
+# - The food's description
+# - The food's Brand Owner
+# - The food's ingredients
+
+require "rails_helper"
+
+RSpec.describe "Food Search" do
+  describe "happy path" do
+    it "allows user to search for food item" do
+      visit root_path
+
+      fill_in :q, with: "sweet potatoes"
+      click_button "Search"
+
+      expect(current_path).to eq(foods_path)
+      expect(page.status_code).to eq(200)
+      expect(page.all('ul', count: 10))
+
+      expect(page).to have_content("Total matching results")
+      expect(page).to have_content("Description")
+      expect(page).to have_content("Brand Owner")
+      expect(page).to have_content("Ingredients")
+      expect(page).to have_content("sweet potatoes")
```

Comment:
Solid testing here. Might be nice to also check for the first and last Food object data we expect to see on the page.